src/components/gas/GasSpeedButton.js

```diff
@@ -230,9 +227,21 @@ const GasSpeedButton = ({
     asset,
     shouldOpenCustomGasSheet.focusTo,
     speeds,
-    openCustomOptions,
   ]);
 
+  const openCustomOptions = useCallback(
+    focusTo => {
+      if (ios) {
+        setShouldOpenCustomGasSheet({ focusTo, shouldOpen: true });
```

Comment:
just talked with @osdnk this was a nasty hack to make some sheet animations work, makes sense to remove it if it fixes android